final/conclusion.tex

```diff
@@ -1,6 +1,36 @@
 \chapter{Conclusion}
 \label{cha:conclusion}
 
+Over the course of the last quarter our project team has explored the Spoofax
+language workbench and created a read-eval-print-loop that operates with any
+language defined in the Spoofax workbench.
+
+During the research phase of the project it became apparent that the project
+team needed to gain quite extensive knowledge about both the conceptual ideas
+of language transformation and the Spoofax API implementing these concepts
+before a successful product would become feasible. Due to this knowledge gap,
+development progress has unfortunately been slower than anticipated for several
+features.
```

Comment:
I agree, this is better left out of the report. We can mention it to Eelco tomorrow, but that's it.